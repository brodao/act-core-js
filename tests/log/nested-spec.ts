import { ACT } from '../../lib';

const { wrap } = require('@brodao/act-jest-snapshot-console');

test('Mensagem aninhada', () => {
	expect(
		wrap(() => {
			ACT.logger.nested('Mensagem aninhada', ['linha 1', 'linha 2', 10]);
		})
	).toMatchSnapshot();
});
