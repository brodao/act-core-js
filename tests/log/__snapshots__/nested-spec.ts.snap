// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mensagem aninhada 1`] = `
Object {
  Symbol(act-jest-snapshot-raw): undefined,
  Symbol(act-jest-snapshot-console): "",
}
`;
